aspnet/publishing/iis.rst

```diff
@@ -34,15 +48,28 @@ With these commands in the *project.json* file, the publish folder will contain
 6. On the **Settings** tab you can select the configuration, target DNX version and publish options. Currently your deployment server must have .NET 4.5.1 or higher to use DNX core. We hope to have a native module in the future that will allow you to use DNX core in the app pool regardless of the full CLR. If you select **Precompile during publishing**, the *Publish\\approot\\src* directory and source files will not be created. If you don't check **Precompile during publishing**, your source files will be found in the  *Publish\\approot\\src* directory. Click **Next**.
```

Comment:
This is obsolete. HttpPlatformHandler is the required native module, .NET 4.5.1 should no longer be required to boot CoreCLR. You should be able to set the app pool to "No managed code" because we run in a separate process and manage the runtime ourselves.